### ISSUE TITLE
[Feature request] Accept rough dates, such as only month or year data for date selectors

### ISSUE THREAD

**Issue by Xalaxis** [AmruthPillai/Reactive-Resume#455]
This would allow for entry of 00/01/2020 - 00/01/2020 for instance.

**Comment by profdrskipper** [comment]
yes please. this would be super helpful. For example, I would just like to specify education/work experience for 2014-2017 or 2014-Present.   I remember that this used to be possible in an older build of rxresume, but the implementation has been changed now with the newer build.
